service/service.py

```diff
@@ -80,6 +80,43 @@ def create_recommendations():
                              'Location': location_url
                          })
 
+######################################################################
+# DELETE A RECOMMENDATION
+# HTTP DELETE /recommendations/{rec_id} - deletes a recommendation record in the database
+######################################################################
+
+@app.route('/recommendations/<int:rec_id>', methods=['DELETE'])
+def delete_recommendations(rec_id):
+    """
+    Delete a recommendation
+
+    This endpoint will delete a recommendation based the id specified in the path
+    """
+    app.logger.info('Request to delete recommendation with id: %s', rec_id)
+    recommendation = Recommendation.find(rec_id)
+    if recommendation:
+        recommendation.delete()
+    return make_response('', status.HTTP_204_NO_CONTENT)
+
+######################################################################
+# UPDATE AN EXISTING RECOMMENDATION
+# HTTP PUT /recommendations/{rec_id} - updates a recommendation record in the database
+######################################################################
+@app.route('/recommendations/<int:rec_id>', methods=['PUT'])
+def update_recommendations(rec_id):
+    """
+    Update a Recommendations
+    This endpoint will update a Recommendation based the body that is posted
+    """
+    app.logger.info('Request to update recommendation with id: %s', rec_id)
+    check_content_type('application/json')
+    recommendation = Recommendation.find(rec_id)
+    if not recommendation:
+        raise NotFound("Recommendation with id '{}' was not found.".format(rec_id))
```

Comment:
I think this line is not covered in the test, could you please write a test for that? By the way you can run `coverage report -m` to see which lines are missed.
I know you may think this is trivial but it's a good practice to cover all branches.

Reply:
Is it okay to cover two issues with one pull request? I think for now we can just re-purpose the existing "update" issue and it should be fine right?

Reply:
Thank you for your comment! Have added another test.